### PR TITLE
Enabled the option to tranlate formio forms with minimal translations

### PR DESCRIPTION
**Pull request by Kevin-Snijders**
For one of the our projects we really need to be able to translate Formio forms which is currently not available. Formio already has a feature to translate things. Currently I have added only add some default translations which could increase in the feature. One of the downsides of the Formio translation is that they use a label as translation so implementation should probably always have some translations as well.

### CODE REVIEW

File: projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts [hunk 1]
```diff
@@ -27,7 +30,23 @@ export class FormioBuilderComponent implements OnInit {
   // eslint-disable-next-line @angular-eslint/no-output-native
   @Output() change: EventEmitter<any> = new EventEmitter();
 
-  constructor() {}
+  readonly currentLanguage$ = this.translateService
+    .stream('key')
+    .pipe(map(() => this.translateService.currentLang));
+
+  readonly formioOptions$: Observable<{language: string, i18n: any }> = this.currentLanguage$.pipe(
```
Comment (mritense) [inline projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts:37]: I think there exists a typing exported by form.io itself?

File: projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts [hunk 2]
```diff
@@ -27,7 +30,23 @@ export class FormioBuilderComponent implements OnInit {
...
+    map(language =>  {
+      const formioTranslations = this.translateService.instant('formioTranslations');
+      return typeof formioTranslations === 'object' ? {
+        language,
+        i18n: {
+          [language]: formioTranslations
+        }
+      } : null;
```
Comment (mritense) [inline projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts:45]: it doesn't break when null is returned?

Comment (Kevin-Snijders) [reply projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts:45]: No null is also valid (won't add any options) and will still work without any warning or error.

File: projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts (hunk 1 above)
Comment (Kevin-Snijders) [reply projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts]: I have updated the the type with a default variant from form.io

**Review by mritense — APPROVED** [review]

**Comment by Kevin-Snijders** [comment]
While writting the docs I found a small bug which has been fixed (also tested it with null instead of options)

**Review by mritense — DISMISSED** [review]

**Review by mritense — APPROVED** [review]

**Review by mritense — APPROVED** [review]
